### DISCUSSION TITLE
原创文章 - 研究 Aptos、Sui 的私钥替换机制

### DISCUSSION THREAD

**Discussion by geometryolife** [starcoinorg/web3startrek#115] [Call for Article]
Move 生态技术文章

任务描述：撰写原创文章+直播讲解文章

题目：
研究 Aptos、Sui 的私钥替换机制。

要求：
1. 阐述清楚原理（原生实现）；
2. 操作实践；
3. 方案对比。

任务奖励：70U + MoveFuns 100积分
首发：DAOrayaki 策展
任务跟进：Joe Chen

**Comment by guijunchen** [comment]
我试试

**Reply by ShirleneLiu** [reply]
蹲一个五一节~
